src/bvals/boundary_conditions.cpp

```diff
@@ -198,6 +214,56 @@ bool DoPhysicalBoundary_(const BoundaryFlag flag, const BoundaryFace face,
 
   return true; // reflect, outflow, user, dims correct
 }
+
+void ProlongateGhostCells_(std::shared_ptr<MeshBlockData<Real>> &rc,
+                           const NeighborBlock &nb, int si, int ei, int sj, int ej,
+                           int sk, int ek) {
+  std::shared_ptr<MeshBlock> pmb = rc->GetBlockPointer();
+  auto &pmr = pmb->pmr;
+
+  for (auto cc_var : rc->GetCellVariableVector()) {
+    if (!cc_var->IsAllocated()) continue;
+    if (!(cc_var->IsSet(Metadata::Independent) || cc_var->IsSet(Metadata::FillGhost)))
+      continue;
+
+    // TODO(LFR): Is this indexing correct for 5 and 6 dimensional fields?
+    int nu = cc_var->GetDim(4) - 1;
+    pmr->ProlongateCellCenteredValues(cc_var.get(), 0, nu, si, ei, sj, ej, sk, ek);
```

Comment:
I think we are, because (if I'm not missing sth) `ProlongateCellCenteredValues` does calculate the other dims:
```
  info_h(b).Nt = var->GetDim(6);
  info_h(b).Nu = var->GetDim(5);
  info_h(b).Nv = var->GetDim(4);
```
It actually looks like ``0, nu` are not even used any more.
How about removing those to clean this up?

Reply:
Correct. When I updated those calls, I moved to pulling this information directly from the variable. The `0` and `nu` were left in to keep the API fixed, but we could remove them from `ProlongateCellCenteredValues` and probably should.